MagneticField/ParametrizedEngine/src/BFit3D_data.h

```diff
@@ -3,1111 +3,1210 @@
-   { -1.37160049098419e-13, -1.60239120526269e-13, -1.21405011287663e-13, -1.72031444267852e-13 },
-   { -2.78485044107995e-13, -1.86164861399683e-13, -1.72305859347797e-13, -8.48161488625391e-14 },
-   { -9.06128559055311e-13, -9.57042583500525e-13, -1.53993701669245e-12, -1.05705062659504e-12 },
-   { -2.77051514938069e-13, -4.33052991402220e-13,  1.85464933620595e-14, -6.70225201972448e-13 },
-   { -5.56718715477705e-14,  4.06127027534827e-13,  4.33412758861253e-13, -5.89114204138403e-14 },
-   { -1.60048047790477e-13, -8.36222188664552e-13, -5.65232045053882e-13, -2.32099118828223e-13 },
-   { -3.37496095895197e-13, -5.80436594480359e-13,  7.58259784113456e-13, -4.97029993732703e-13 },
-   { -4.57730678585326e-13,  3.27788632586418e-13, -9.59462491671028e-13, -6.96884485137794e-13 },
-   {  6.30057143635801e-14, -6.80165804263083e-13, -9.97946951785042e-17, -1.82341973199206e-14 },
-   {  2.74337368489962e-12,  1.68263816504159e-12,  3.55425087180429e-12,  2.95173684158353e-12 },
-   {  1.70330857721202e-12,  2.85474422703727e-12,  1.24845382260745e-12,  2.28130498151709e-12 },
-   {  3.45322033109625e-12,  3.10550577747037e-12,  6.78767521979236e-12,  4.01091205539599e-12 },
-   { -3.02898205793240e-13,  5.12451263236909e-13, -1.43183122409818e-12, -5.49243100690923e-13 },
-   {  1.16936321379766e-12,  1.37653136352337e-12,  9.85522850706541e-13, -8.24638787198278e-13 },
-   { -8.64374675972936e-13,  7.04178200449449e-13, -4.92466744031305e-12, -9.32110850129566e-13 },
-   { -9.59621816641773e-12, -1.39753150906529e-11, -2.10080890996816e-11, -2.05561142050894e-11 },
-   { -5.63379531000104e-12, -7.01765369661281e-12, -8.38668573911187e-12, -1.73513734865394e-11 },
-   { -4.07454373599104e-11, -4.81950837126463e-11, -2.27080114271624e-11, -2.86820952560172e-11 },
-   {  6.95489644198036e-11,  7.27626259095232e-11,  6.70231262012377e-11,  9.09463504257230e-11 },
-   { -3.08547002927579e-11, -1.72155939101793e-11, -8.28971557995200e-12,  8.09949768447610e-11 },
-   { -7.54417212424961e-11, -2.23512915439469e-11, -4.65013336868375e-11, -9.27361830701006e-11 },
-   { -2.93731718679558e-11, -4.76191500727179e-10,  1.96485963733147e-10, -3.24729386108877e-10 },
-   { -1.48512347391017e-09, -1.38427377797602e-09, -1.65928813179673e-09, -1.53201004986758e-09 },
-   { -6.81760689438126e-09, -9.41618299931884e-09, -9.96460624244661e-09, -6.03558463299611e-09 },
-   {  1.73905380895445e-09,  1.12852499668810e-10,  1.00764552541746e-09, -9.11673757184537e-10 }
-}; //BFit3D::C0[360][4]
+  //Spline coefficients for the field expansion coefficients
+  //_______________________________________________________________________________
+  const double BFit3D::C0[360][4] = {  //Constant terms, shifted to left
+                                       //(since at B_nom = 0 all of them are equal to zero)
+      {2.02156144404716e+00, 3.51621670097246e+00, 3.81142532261878e+00, 4.01241090152589e+00},
```

Comment:
Definitely the previous version was much more readable.
This is probably a god place where to forbid clang formatting.